core/views/coreuser.py

```diff
@@ -250,10 +252,12 @@ def get_permissions(self):
                                'reset_password',
                                'reset_password_check',
                                'reset_password_confirm',
-                               'invite_check']:
+                               'invite_check', 'update', 'partial_update']:
```

Comment:
@ashishkmishra36 I have a query here, can you check this and confirm with Greg on whether update be allowed for the user themselves.

Reply:
Implemented update profile as a separate endpoint